source/extensions/filters/listener/proxy_protocol/proxy_protocol.h

```diff
@@ -118,7 +122,7 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   bool parseV2Header(char* buf);
   absl::optional<size_t> lenV2Address(char* buf);
 
-  Network::ListenerFilterCallbacks* cb_{};
+  Network::ListenerFilterCallbacks* cb_{nullptr};
```

Comment:
nit: the original initialization with `{}` is equivalent.  Consider reverting this change.

Reply:
got it, thanks